frontend/src/scenes/App.tsx

```diff
@@ -19,6 +19,20 @@ import { router } from 'kea-router'
 import { CommandPalette } from 'lib/components/CommandPalette'
 import { UpgradeModal } from './UpgradeModal'
 import { teamLogic } from './teamLogic'
+import posthog from 'posthog-js'
```

Comment:
I'm not sure this is what we want - a lot of the current code is already relying on window.posthog and this updates a small piece of it.

Reply:
Yep, have changed all the other instances of window.posthog